keyboards/tominabox1/slab/keymaps/default/keymap.c

```diff
@@ -0,0 +1,77 @@
+/* Copyright 2023 tjcampie
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+#include "analog.h"
```

Comment:
In general, the default keymap should be kept fairly clean, as it's what's copied for new keymaps, and this is ... rather complicated for a newer user. 

it may be a better idea to use a custom matrix here and add an "up/down" position for the switches in the switch matrix.

Reply:
I'm not sure what you mean here. This board is really targeted for Vial compatibility so I don't know if simplification is too necessary